tests/test_user_model.py

```diff
@@ -61,9 +61,12 @@ def test_is_admin(self):
     def test_is_eligible_for(self):
         another_task_type = 'task_type_2'
         u1 = User(username='1', email='1@email.com', admin=True).save()
+        u2 = User(username='2', email='2@email.com', admin=False).save()
```

Comment:
Would you mind, good sir, spending your precious time to split this certain test case into two separate ones, that will bear distinctive names?